src/composables/events/GitEvent.js

```diff
@@ -18,6 +18,12 @@ const FetchEvent = new Subject();
  */
 const PullEvent = new Subject();
 
+/**
+ * Represent a rxjs Event object to emit and to receive events when a push is done on git.
+ * @typedef {Subject} FetchEvent
+ */
+const PushEvent = new Subject();
```

Comment:
I would have put this commit first, and then the commit that uses this event after